acme/acme/messages.py

```diff
@@ -92,10 +93,10 @@ def code(self):
             return code
 
     def __str__(self):
-        return ' :: '.join(
-            part for part in
+        return b' :: '.join(
+            part.encode('ascii', 'backslashreplace') for part in
```

Comment:
I also spent some time trying to figure out a quick fix to nicely print the Unicode characters. It'd be good if we could do this because I suspect the vast majority of the time the user's terminal supports UTF-8 and simply printing the Unicode characters would work. There are times it won't though so I think just always replacing the characters is fine for now. I think the real solution is properly handling encoding output/failures, but this would require changes all over the code base.

I agree `backslashreplace` is the better option here.